controllers/clusters/helpers.go

```diff
@@ -159,6 +161,38 @@ func getSortedAppVersions(versions []*models.AppVersions, appType string) []*ver
 	return nil
 }
 
+func fetchMaintenanceEventStatuses(
+	api instaclustr.API,
+	clusterID string,
+) ([]*clusterresourcesv1beta1.ClusteredMaintenanceEventStatus,
+	error) {
+
```

Comment:
How about making it as a method for `instaclustr.API` client?

Reply:
That sounds pretty logical, will move to the client